pkg/kubectl/cmd/wait/wait.go

```diff
@@ -61,6 +62,9 @@ var (
 		kubectl wait --for=delete pod/busybox1 --timeout=60s`)
 )
 
+// ErrNoMatchingResources is returned when there is no resources matching a query.
+var ErrNoMatchingResources = errors.New("failed to find matching resources")
```

Comment:
var -> const
If this is used only in this package you can lowercase

Reply:
> If this is used only in this package you can lowercase

Good idea, thanks. I've made it to be unexported.

> var -> const

I think, the go compiler will complain about `errors.New("something")` not being a constant. I've stolen the approach from other packages where errors are defined as vars using `errors.New()`. Few examples:

https://github.com/kubernetes/kubernetes/blob/d2721eccd9272f5330765b4d3610f1b9a224e470/pkg/client/conditions/conditions.go#L30

https://github.com/kubernetes/kubernetes/blob/d2721eccd9272f5330765b4d3610f1b9a224e470/pkg/cloudprovider/providers/openstack/openstack.go#L62-L68